python/lsst/source/injection/bin/ingest_injection_catalog.py

```diff
@@ -27,11 +27,16 @@
 
 from astropy.table import Table
 from lsst.daf.butler import Butler
+from lsst.daf.butler.formatters.parquet import arrow_to_astropy, pq
 
 from ..utils import ingest_injection_catalog
 from .source_injection_help_formatter import SourceInjectionHelpFormatter
 
 
+def _is_parquet(filename: str):
+    return filename.endswith((".parquet", ".parq"))
```

Comment:
I do not like this method at all. It seems very scenario-specific, and probably not the way we want to go.

Reply:
But, sadly, it seems as though the astropy folks also do a similar thing, so we may be forced to do the same here... https://github.com/astropy/astropy/blob/b447aed5d7237aa6cded5cd6204ee5cbf8c20a5e/astropy/io/misc/parquet.py#L54

Reply:
This is what @erykoff recommended...

The alternatives are to use [the daf_butler default_extension](https://github.com/lsst/daf_butler/blob/86eadf3d0e5b13c4f93b94cb6ad73695a375c3d7/python/lsst/daf/butler/formatters/parquet.py#L88) (which by default doesn't accept `.parquet`), or to actually open the file and check if it's parquet via [astropy](https://github.com/astropy/astropy/blob/b447aed5d7237aa6cded5cd6204ee5cbf8c20a5e/astropy/io/misc/parquet.py#L26), which notably, checks for the exact same extensions if you only give it a filename.